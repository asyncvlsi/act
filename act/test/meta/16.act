defptype mytype(pint a, b);

template<mytype x>
defproc test(bool i[x.a]; bool o[x.b])
{
  i[0] = o[0];
  { x.a + 1 = x.b };
}

mytype q;

q.a = 5;
//q.b = 6;

test<q> w;
