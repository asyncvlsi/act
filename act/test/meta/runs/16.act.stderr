In expanding test<q> (16.act:3)
In expanding ::<Global>
Error on or near line number 15.
 id: q
FATAL: PStruct has an uninitialized pint @ position 1
